src/reflect/scala/reflect/internal/Definitions.scala

```diff
@@ -1578,6 +1578,10 @@ trait Definitions extends api.StandardDefinitions {
 
     /** Efficient access to member symbols which must be looked up each run. Access via `currentRun.runDefinitions` */
     final class RunDefinitions {
+      // until we bootstrap and they are implemented as `macro ???` to trigger fasttrack directly
+      def treatLikeMacro(sym: Symbol) =
+        sym.isMethod && ((sym.name == nme.s || sym.name == nme.raw_) && sym.owner == StringContextClass) && (sym == StringContext_s || sym == StringContext_raw)
```

Comment:
Why not just:

```
def treatLikeMacro(sym: Symbol) = (sym eq StringContext_s) || (sym eq StringContext_raw)
```

Reply:
This method seemed to be hit a lot.

Reply:
(Early on in type checking, so didn’t want to force symbols when not needed.)